src/modules/10-common/components/MultiTypeSelect/MultiTypeSelect.tsx

```diff
@@ -36,7 +36,7 @@ export interface MultiTypeSelectProps {
   className?: string
   name: string
   label: string | React.ReactElement
-  formik?: FormikContext<any>
+  formik?: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
  formik?: FormikContextType<unknown>
```